receiver/prometheusexec/README.md

```diff
@@ -0,0 +1,126 @@
+# This receiver is currently under development, do not use.
+// TODO: Remove above message when done
+
+# Prometheus_exec Receiver
+
+This receiver makes it easy for a user to collect metrics with the Collector from third-party services such as MySQL, Apache, Redis, etc. It's meant for people who want a plug-and-play solution to getting metrics from those third-party services that don't natively export metrics or speak any instrumentation protocols. Through the configuration file, you can indicate which binaries to run (usually [Prometheus exporters](https://prometheus.io/docs/instrumenting/exporters/), which are custom binaries that will expose the services' metrics to the Prometheus protocol). It supports starting binaries with flags and environment variables, retrying them with exponentional backoff if they crash, string templating, and random port assignments. If you do not need to spawn the binaries locally, please condider using the [core Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/prometheusreceiver) or the [Simple Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver/simpleprometheusreceiver).
+
+## Overview
+Here's an example Collector configuration file that starts two `prometheus_exec` receivers and two exporters.
```

Comment:
I think you can probably limit this README to just focus on the `prometheus_exec` config. The rest of the config is already well documented in the general Collector docs.

Reply:
Okay, I'll remove! thanks